tests/unit/model/test_model_item.py

```diff
@@ -18,20 +18,50 @@
 
 import pytest
 
+from structurizr.model import Model, SoftwareSystem, SoftwareSystemIO
 from structurizr.model.model_item import ModelItem
 
 
+@pytest.fixture(scope="function")
+def model() -> Model:
```

Comment:
Please provide a docstring for this fixture.

```suggestion
def empty_model() -> Model:
```

Reply:
Done.